torchtnt/utils/rank_zero_log.py

```diff
@@ -43,7 +43,7 @@ def rank_zero_warn(
     if get_global_rank() != 0:
         return
     logger = logger or _LOGGER
-    logger.warn(*args, stacklevel=2, **kwargs)
+    logger.warning(*args, stacklevel=2, **kwargs)
```

Comment:
thanks!

the test for this needs to be updated now as well:

https://github.com/pytorch/tnt/blob/535dc56887b7b574878b14a14ce6cb37c4c26741/tests/utils/test_rank_zero_log.py#L33-L53

Reply:
should the fn be renamed `rank_zero_warning` ? Python has deprecated `logging.warn` in favor of `logging.warning` in 3.3.